File: ljh/java/src/level1/test/SumOfDivisorTest.java
package level1.test;

import static org.junit.Assert.assertEquals;

import org.junit.Before;
import org.junit.Test;

import level1.SumOfDivisor;

public class SumOfDivisorTest {
	private int[] n;
	private int[] answers;
	private SumOfDivisor sod;
	
	@Before
	public void setUp() throws Exception {
		n = new int[] {12, 5};
		answers = new int[] {28, 6};
		sod = new SumOfDivisor();
	}
	
	@Test
	public void test() {
		for(int i=0; i<answers.length; i++) {
			assertEquals(sod.solution(n[i]), answers[i]);
		}
	}
}

File: ljh/javascript/level1/test/SumOfDivisor.test.js
import SumOfDivisor from '../SumOfDivisor';
test(`약수의 합`, () => {
    expect(new SumOfDivisor().solution(12)).toBe(28);
});
test(`약수의 합`, () => {
    expect(new SumOfDivisor().solution(5)).toBe(6);
});